.github/release-drafter-cli.yml

```diff
@@ -1,37 +1,7 @@
+_extends: guardrail:.github/release-drafter.yml
```

Comment:
if you want we can be specific that it should be `
guardrail-dev/guardrail:.github/release-drafter.yml`